feat: add Sync methods (#371)
guardian/consent-management-platform: README.md

    );
```

### `cmp.willShowPrivacyMessageSync()`

returns: `Boolean`

> Returns `true` if the CMP will show the initial privacy message (or is, or was), or `false` if not.

Reviewer: ```suggestion
Returns `true` if the CMP will show the initial privacy message, or `false` if not.
```

Author: how come?

Reviewer: It’s the triple-“or” that’s confusing.

> `1` if `A` (or `B`, or `C`), or `0` if not.

Could we make it as below?

```suggestion
Returns `true` if the CMP has shown, is showing or will show the initial privacy message. Returns `false` otherwise.
```

I don’t think too strongly about this in any case.
